packages/storybook/src/nimble/anchor-tabs/anchor-tabs.stories.ts

```diff
@@ -27,6 +29,49 @@ interface ToolbarArgs {
     toolbar: boolean;
 }
 
+const simpleTabs = [
+    {
+        title: 'Google',
+        id: '1',
+        disabled: true,
+        href: 'https://www.google.com'
+    },
+    { title: 'NI', id: '2', disabled: false, href: 'https://www.ni.com' },
+    { title: 'Nimble', id: '3', disabled: false, href: 'https://nimble.ni.dev' }
+] as const;
+
+const wideTabs = [
+    {
+        title: 'Tab 1 that is too long to fit in the drop down width',
+        id: '1',
+        disabled: false,
+        href: 'https://nimble.ni.dev'
+    },
+    {
+        title: 'Tab 2 that is also too long but disabled',
+        id: '2',
+        disabled: true,
+        href: 'https://nimble.ni.dev'
+    },
+    { title: 'Short', id: '2', disabled: false, href: 'https://nimble.ni.dev' }
+] as const;
+
+const manyTabs: AnchorTabArgs[] = [];
+for (let i = 0; i < 100; i++) {
+    manyTabs.push({
+        title: `Tab ${i}`,
+        id: `${i}`,
+        disabled: false,
+        href: `https://nimble.ni.dev?tabid=${i}`
+    });
+}
```

Comment:
@atmgrifter00 it's still inconsistent Remember that the activeid parameter can be used with each of the configurations. It seems like 1-indexing is preeferred so lets do it here to be consistent

```suggestion
for (let i = 1; i <= 100; i++) {
    manyTabs.push({
        title: `Tab ${i}`,
        id: `${i}`,
        disabled: false,
        href: `https://nimble.ni.dev?tabid=${i}`
    });
}
```